api/webhooks/webhooks.py

```diff
@@ -88,6 +92,12 @@ def trigger_sample_webhook(
     return _call_webhook(webhook, serializer.data)
 
 
+@backoff.on_exception(
+    backoff.expo,
+    requests.exceptions.RequestException,
+    max_tries=8,
```

Comment:
8 seems excessive to me? I would say 3 or 5 would be sufficient?

Reply:
Yes, I was not sure what to use. While testing - I found 3 to be quite fast, so I think 5 would be better.